### PR TITLE
Added Fregoli Delusion

### PR DESCRIPTION
**Pull request by rudr4x5h**
I've made the following changes to `readme.md` file:
- Fixed trailing whitespaces.
- Added **Fregoli Delusion** to the *Psychological Syndromes* section.
- Updated the table of contents.

@icyflame kindly review.

### CODE REVIEW

**Comment by Megha66** [comment]
fregoli delusion is interesting. nice entry.

**Comment by rudr4x5h** [comment]
Thanks @Megha66. Indeed it is **one of the many** interesting syndromes out there!

File: readme.md [hunk 1]
```diff
@@ -176,6 +177,17 @@ the anecdote, you can throw away the data."
 
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
+#### [Fregoli Delusion](https://en.wikipedia.org/wiki/Fregoli_delusion)
+
+> Belief that different people are in fact the same!
+
+Its a rare disorder in which a person holds a delusional belief that different
```
Comment (icyflame) [inline readme.md]: apostrophe

File: readme.md [hunk 2]
```diff
@@ -176,6 +177,17 @@ the anecdote, you can throw away the data."
...
+people are in fact a single person who changes appearance or is in disguise!
+The syndrome may be related to a brain lesion and is often of a paranoid nature,
+with the delusional person believing themselves persecuted by the person they
+believe is in disguise.
+
+[R: Fregoli Delusion](https://www.ncbi.nlm.nih.gov/pubmed/25219764)
```
Comment (icyflame) [inline readme.md]: This is a paper, that isn't ideal. Please find two other references.

**Review by icyflame — CHANGES_REQUESTED** [review]

**Comment by rudr4x5h** [comment]
@icyflame done! kindly review.

**Review by icyflame — APPROVED** [review]

**Pull request closed without merging**